host/onebox.go

```diff
@@ -146,10 +152,26 @@ func (c *cadenceImpl) HistoryServiceAddress() []string {
 	return hosts
 }
 
+func (c *cadenceImpl) HistoryPProfPort() []int {
+	ports := []int{}
+	startPort := 7300
+	for i := 0; i < c.numberOfHistoryHosts; i++ {
+		port := startPort + i
+		ports = append(ports, port)
+	}
+
+	c.logger.Infof("History pprof prots: %v", ports)
```

Comment:
nit: typo prots->ports

Reply:
Do we want to log this? Other ones aren't being logged.

Reply:
the server, when starts, actually shows the address and port:
INFO[2017-12-21T15:23:09-08:00] Created RPC dispatcher for 'cadence-matching' and listening at '127.0.0.1:7935' 
INFO[2017-12-21T15:23:09-08:00] Created RPC dispatcher for 'cadence-frontend' and listening at '127.0.0.1:7933' 
INFO[2017-12-21T15:23:09-08:00] Created RPC dispatcher for 'cadence-history' and listening at '127.0.0.1:7934'

Reply:
typo updated